transactions.md

```diff
@@ -145,6 +153,10 @@ transactions are never meant to be used.
 
 The Emergency `scriptPubKey` is not known to the managers.
 
+In addition to the deposit output, the Emergency transaction features a data-carrying output committing
+to the Deposit transaction it's used for in order for the `ANYONECANPAY | ALL` signature to not be replayed
```

Comment:
"...committing to the Deposit transaction id _and the output index (vout)_ it is used for..."